templates/block-templates/email-template.html

```diff
@@ -12,7 +12,7 @@
 				style="background-color:#ffffff;margin-top:0px;margin-bottom:8px;padding-top:40px;padding-right:56px;padding-bottom:0px;padding-left:56px">
 				<!-- wp:site-logo {"width":48,"className":"inline-block","style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}}}} /-->
 
-				<!-- wp:site-title {"textAlign":"left","style":{"color":{"text":"#101517"},"typography":{"fontSize":"16px","fontStyle":"normal","fontWeight":"700"},"elements":{"link":{"color":{"text":"#101517"}}},"spacing":{"margin":{"top":"20px","right":"0px","bottom":"0px","left":"0px"},"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}}},"className":"inline-block"} /-->
+				<!-- wp:site-title {"textAlign":"left","isLink":false,"style":{"color":{"text":"#101517"},"typography":{"fontSize":"16px","fontStyle":"normal","fontWeight":"700"},"elements":{"link":{"color":{"text":"#101517"}}},"spacing":{"margin":{"top":"20px","right":"0px","bottom":"0px","left":"0px"},"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}}},"className":"inline-block"} /-->
```

Comment:
Is it disabling the site link? I think it is important when the blog doesn't have a logo.